### DISCUSSION TITLE
v2.0.0-rc2

### DISCUSSION THREAD

**Discussion by PoeppingT** [awslabs/aws-saas-boost#292] [Announcements]
Hello curious reader! 🚀 This release marks the third and final release candidate for the v2 multi-container release of SaaS Boost. This release candidate contains loads of bugfixes to bring the highest quality bar for multi-container, as well as some new features: a new Tier UI for easy use in the Admin UI, enhancements to the Installer for v2, and new support for FSX ONTAP as a filesystem in applications. 

A very special thank you to those members of our community that have participated in the pre-release discussions for multi-container: both kicking the tires and submitting bug reports, and congratulations to two new contributors: @AditModi and @muylucir ! We hope you stick around 😄 

As before, the discussion for v2.0.0 release candidates continues in the discussion: https://github.com/awslabs/aws-saas-boost/discussions/206

## What's Changed
* Fix unit test to be OS platform independent. Clean up PowerShell script by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/181
* Fix Java sample app dependency clashes. Don't use deprecated Spring by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/183
* java.nio.file.Path uses OS dependent file separator character but we by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/184
* Resolve OS dependent path separators in S3 keys for web app files by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/189
* Fix small bugs with install/delete for Metrics module. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/187
* Bump postgresql from 42.2.13 to 42.2.25 in /resources/custom-resources/rds-bootstrap by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/192
* Bump postgresql from 42.2.13 to 42.2.25 in /samples/java by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/193
* Bump node-fetch from 2.6.1 to 2.6.7 in /.github/actions/prlinter by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/195
* Bump postgresql from 42.2.25 to 42.3.3 in /samples/java by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/197
* Bump postgresql from 42.2.25 to 42.3.3 in /resources/custom-resources/rds-bootstrap by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/198
* Update tenant-onboarding.yaml by @AditModi in https://github.com/awslabs/aws-saas-boost/pull/214
* Add multi-container support. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/221
* Bump spring-boot-starter-web from 2.6.1 to 2.6.6 in /samples/java by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/224
* Bump jackson-databind from 2.11.2 to 2.12.6.1 in /installer by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/222
* Bump jackson-databind from 2.11.0 to 2.12.6.1 in /layers/utils by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/223
* Fix workload deploy for service names with capitalized letters by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/226
* Enabling SQS encryption at rest using the default KMS key by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/229
* Making resource names unique per application service by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/236
* Add Tier UI and implement default tiers. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/228
* Fill SSL Certificate options from ACM in Admin UI AppConfig. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/230
* Fixing the .NET Framework sample to work with v2 by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/238
* Doc updates for v2 by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/237
* Fix MSBuild for .NET Framework Sample App by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/244
* RDS Bootstrap fixes by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/251
* Fix bug selecting tier for provision tenant and creating a new defaul… by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/249
* Move database configuration to services, performance characteristics (instanceClass) to tier. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/239
* Simplify the filesystem JSON in AppConfig. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/248
* Fix HTTP and HTTPS listener rules on load balancer by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/260
* Support for FSx ONTAP during tenant onboarding. by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/258
* Allow for EC2 Launch Type with Linux-based services. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/259
* Support FSX Ontap in AppConfig. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/261
* Service shutdown and startup v2 by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/268
* Default ecsLaunchType to EC2 for Windows based services by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/270
* Prevent CloudFormation parameter errors when using EFS by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/271
* Bump postgresql from 42.3.3 to 42.4.1 in /samples/java by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/265
* Bump postgresql from 42.4.0 to 42.4.1 in /resources/custom-resources/rds-bootstrap by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/266
* Allow dash in SaaS Boost environment name by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/274
* CloudFormation custom resource response retry by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/275
* Bump @actions/core from 1.2.7 to 1.9.1 in /.github/actions/prlinter by @dependabot in https://github.com/awslabs/aws-saas-boost/pull/278
* Add UI support for selecting FSX Ontap for Windows Applications. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/263
* Fix Issue 280 by @brtrvn in https://github.com/awslabs/aws-saas-boost/pull/281
* Add Tier to tag setting by @muylucir in https://github.com/awslabs/aws-saas-boost/pull/276
* Change documentation to include FSx ONTAP support. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/264
* Fix installing multiple environments in one account by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/283
* Bring Utils into installer to enable intelligent installer update by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/284
* Fix installer delete for v2 by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/279
* Fix installing SaaS Boost without first running a mvn build. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/285
* Fixes #287. EFS and FSX_WINDOWS filesystems were not sending type in … by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/289
* Fix an issue in the install script where having no region set in the CLI errors unhelpfully. by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/291
* Installer update enhancements by @PoeppingT in https://github.com/awslabs/aws-saas-boost/pull/288

## New Contributors
* @AditModi made their first contribution in https://github.com/awslabs/aws-saas-boost/pull/214
* @muylucir made their first contribution in https://github.com/awslabs/aws-saas-boost/pull/276

**Full Changelog**: https://github.com/awslabs/aws-saas-boost/compare/v1.1.2...v2.0.0-rc2

<hr /><em>This discussion was created from the release <a href='https://github.com/awslabs/aws-saas-boost/releases/tag/v2.0.0-rc2'>v2.0.0-rc2</a>.</em>

**Comment by tomg-formpak** [comment]
@PoeppingT Thank you. 

We are using version v2.0.0-rc1. Can we update this new release (v2.0.0-rc2) by installing the SaaS Boost again with the option " Update Web Application for existing AWS SaaS Boost deployment"?

**Reply by PoeppingT** [reply]
Hi @tomg-formpak , that option in the installer only updates the Administrative UI for SaaS Boost. Instead use the "Update existing AWS SaaS Boost deployment." option.
